.github/workflows/package.yml

```
@@ -0,0 +1,40 @@
+name: Package
+
+on:
+  push:
+    branches: [ "develop"]
+    paths-ignore:
+     - target/**
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 5

(indentation)